Bottom: 4d0d9bc451ab50d659c9d5e2135416d4296613ce
Top:    f4d869fb656ea58da4e6f58177ae034a894c63a3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-14 11:47:15 -0500

Refresh of all-move-vars-to-defaults

---
diff --git a/roles/docker-openldap/defaults/main.yml b/roles/docker-openldap/defaults/main.yml
index 58247cf..473571b 100644
--- a/roles/docker-openldap/defaults/main.yml
+++ b/roles/docker-openldap/defaults/main.yml
@@ -28,3 +28,31 @@ openldap_backup_files:
 
 # Network names
 docker_network_frontend: 'frontend'
+
+
+# the name of the image being duilt and used for the container
+openldap_image_repo: https://github.com/novatechweb/docker-openldap.git
+openldap_image_name: '{{ docker_registry_username }}/ldap'
+
+# the name of the container being started
+openldap_container_name: '{{ docker_name_prefix }}ldap'
+
+# the name of the data-volume used by the container
+openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
+openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
+
+# restore directories to temporarly store data being restored into docker containers
+openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
+host_openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
+
+
+
+# *****************************************************************************
+# Utility container - phpldapadmin
+
+# the name of the image used for the container
+phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
+phpldapadmin_image_name: '{{ docker_registry_username }}/phpldapadmin'
+
+# the name of the container being started
+phpldapadmin_container_name: '{{ docker_name_prefix }}phpldapadmin'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 125fc4f..55c3e11 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -66,12 +66,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-openldap'
 
-
 - name: build image
   docker_image:
     name: '{{ openldap_image_name }}'
     tag: '{{ docker_image_tag }}'
     path: '{{ docker_projects_dir }}/docker-openldap'
+    force: true
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openldap/vars/main.yml b/roles/docker-openldap/vars/main.yml
index f66b70d..ed97d53 100644
--- a/roles/docker-openldap/vars/main.yml
+++ b/roles/docker-openldap/vars/main.yml
@@ -1,28 +1 @@
 ---
-
-# the name of the image being duilt and used for the container
-openldap_image_repo: https://github.com/novatechweb/docker-openldap.git
-openldap_image_name: '{{ docker_registry_username }}/ldap'
-
-# the name of the container being started
-openldap_container_name: '{{ docker_name_prefix }}ldap'
-
-# the name of the data-volume used by the container
-openldap_cv_name: '{{ docker_name_prefix }}ldap_CV'
-openldap_dv_name: '{{ docker_name_prefix }}ldap_DV'
-
-# restore directories to temporarly store data being restored into docker containers
-openldap_docker_backup_dir: '{{ docker_backup_dir }}/LDAP'
-host_openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
-
-
-
-# *****************************************************************************
-# Utility container - phpldapadmin
-
-# the name of the image used for the container
-phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
-phpldapadmin_image_name: '{{ docker_registry_username }}/phpldapadmin'
-
-# the name of the container being started
-phpldapadmin_container_name: '{{ docker_name_prefix }}phpldapadmin'
